### DISCUSSION TITLE
Problems using Facewallet through Web 3 Modal

### DISCUSSION THREAD

**Discussion by DededeDev** [WalletConnect/walletconnect-monorepo#4307] [Developer Support] [answered]
We are currently creating a DAPP that needs to support multiple wallets through Web 3 Modal.
Wallet Connect, Metamask and Coinbase are all working fine, but we're also trying to enable Facewallet for social logins.

When I click on face wallet it asks me to open up another browser window.

![a](https://github.com/WalletConnect/walletconnect-monorepo/assets/59858313/0c9b474b-0411-47e5-bf2d-af8f8c84683c)

Here I get a modal to connect the app.
The WalletConnect link is automatically pasted and looks fine.
When I click on connect, it claims that a connection has been established, however on my DAPP I don't get logged in and I still see the same screen.
![c](https://github.com/WalletConnect/walletconnect-monorepo/assets/59858313/5e0a10a4-4194-47bd-aedc-6b13f02244a4)

I believe my config is probably not related to this issue, but I'll post it below for reference.

```
export const walletConnectProjectId = process.env.NEXT_PUBLIC_PROJECT_ID || "";

const { chains } = configureChains(
  [sepolia],
  [publicProvider()],
)

export const config = defaultWagmiConfig({
  chains: chains,
  projectId: walletConnectProjectId,
  enableInjected: false, // Delete to show Browser Wallets like brave
})

createWeb3Modal({
  wagmiConfig: config,
  projectId: walletConnectProjectId,
  featuredWalletIds: [
    "b6d18ff342920bb492f810bb070a064d6031ec2c3d6fffecb6ca233c8a591e00", // Facewallet
  ]
})
```

Any advice would be appreciated!

**Comment by glitch-txs** [comment, marked as answer]
Hi, this works for me 
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/66949816/c038aa6b-33f6-4929-8811-308205960aac)

Make sure to use the latest version of Web3Modal v4 with Wagmi v2

**Comment by dot-haechi** [comment]
@DededeDev 
Hello, I'm Dot from FaceWallet team.
I'd like to help you with troubleshooting.
May I know the console log or any other information?
